service/dal/db_handler.py

```diff
@@ -16,3 +16,6 @@ def __call__(cls, *args, **kwargs):
 class DalHandler(ABC, metaclass=_SingletonMeta):
     @abstractmethod
     def create_order_in_db(self, customer_name: str, order_item_count: int) -> Order: ...  # pragma: no cover
+    
+    @abstractmethod
```

Comment:
add cdk code that generates all the required resources, make sure to add at the correct place